CrossPlatform/Pages/TaskDetailsPage.cs

```diff
@@ -16,8 +17,19 @@ public class TaskDetailsPage : BasePage
         readonly Query deleteButton;
         readonly Query doneIndicator;
 
+        protected override Trait Trait
+        {
+            get
+            {
+                return new Trait
+                {
+                    AndroidTrait = x => x.Marked("menu_save_task"),
+                    iOSTrait = x => x.Marked("Task Details")
+                };
+            }
+        }
+
```

Comment:
Yeah, I like option two, but I'm worried that people won't know what's going on since that feature is so new. I guess they can just copy it, but idk